src/objectTranslations/GetStartedChecklistSection__mdt-en_US.objectTranslation

```diff
@@ -13,6 +13,19 @@
         <label><!-- Description Label Name --></label>
         <name>Description_Label__c</name>
     </fields>
+    <fields>
+        <help><!-- To select in which checklist has displayed this section --></help>
```

Comment:
I recommend changing this to "Determines which checklist will display this section". I recommend changing this for all of the translations.